File: PHP/petshop.php
<?php
    // ATRIBUT PRIVATE
    class petshop{
        private $id = '';
        private $nama_produk = '';
        private $harga_produk = 0;
        private $stok_produk = 0;
        private $foto = '';

    // CONSTRUCTOR
    public function __construct($id= 0, $nama_produk = '', $harga_produk=0, $stok_produk=0, $foto = ''){
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->harga_produk = $harga_produk;
        $this->stok_produk = $stok_produk;
        $this->foto = $foto;
    }

    // GETTER
    public function get_id(){
        return $this->id;
    }
    public function get_nama_produk(){
        return $this->nama_produk;
    }
    public function get_harga_produk(){
        return $this->harga_produk;
    }
    public function get_stok_produk(){
        return $this->stok_produk;
    }
    public function get_foto() {
        return $this->foto;
    }
    
    // SETTER
    public function set_id($id){
        $this->id = $id;
    }
    public function set_nama_produk($nama_produk){
        $this->nama_produk = $nama_produk;
    }
    public function set_harga_produk($harga_produk){
        $this->harga_produk = $harga_produk;
    }
    public function set_stok_produk($stok_produk){
        $this->stok_produk = $stok_produk;
    }
    public function set_foto($foto) {
        $this->foto = $foto;
    }
}
?>

File: PYTHON/petshop.py
class petshop:

    # ATRIBUT PRIVATE
    __id = 0
    __nama_produk = ""
    __harga_produk = 0
    __stok_produk = 0

    # CONSTRUCTOR
    def __init__(self, id=0, nama_produk="", harga_produk=0, stok_produk=0):
        self.__id = id
        self.__nama_produk = nama_produk
        self.__harga_produk = harga_produk
        self.__stok_produk = stok_produk

    # GETTER
    def get_id(self):
        return self.__id

    def get_nama_produk(self):
        return self.__nama_produk

    def get_harga_produk(self):
        return self.__harga_produk
    
    def get_stok_produk(self):
        return self.__stok_produk

    # SETTER
    def set_id(self, id):
        self.__id = id

    def set_bahan(self, bahan):
        self.__bahan = bahan

    def set_warna(self, warna):
        self.__warna = warna
